teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher
+  def initialize(age:, name:, specialization:, parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] The Teacher class should inherit from the Person class

- [ ] Kindly refactor your parameters to match the expected set up in the Person class

  ```
  class Teacher < Person
    def initialize(age, specialization, name = "Unknown", parent_permission: true)
      super(age, name, parent_permission: parent_permission)
      ...
  ```